chips/earlgrey/src/registers/alert_handler_regs.rs

```diff
@@ -3,10 +3,10 @@
 //   Apache License, Version 2.0 (LICENSE-APACHE <http://www.apache.org/licenses/LICENSE-2.0>)
 //   MIT License (LICENSE-MIT <http://opensource.org/licenses/MIT>)
 
-// Built for earlgrey_silver_release_v5-11270-gcd74b4221
-// https://github.com/lowRISC/opentitan/tree/cd74b42214fb37ba6b2d5bd4fa13ff0273f77e4e
+// Built for Earlgrey-M2.5.1-RC1-389-g21ce4e9761
```

Comment:
Curious that this isn't `Earlgrey-M2.5.2-RC0`; I think we'll need to update the workspace_status script so this gets the correct tag in the future.

It appears the problem is that `git describe` only wants to use "annotated tags" and the M2.5.2-RC0 tag isn't annotated.  I think we'll have to change the command to something like `git describe --tags` to pick up an un-annotated tag.

We'll keep this as-is for now.  The commit `21ce4e976` _is_ the correct commit.

Reply:
Yes it look like a bug in the `get_workspace_status.sh` script.
Command `git describe --always --tag` return correct string.

Reply:
@cfrantz  This issue should be fixed by this PR:
https://github.com/lowRISC/opentitan/pull/19260

We should also add missing annotations to OT repo tags.